### DISCUSSION TITLE
RUFFLE INCOMPATIBILITY

### DISCUSSION THREAD

**Discussion by CIACIACIACIACIACIA** [ruffle-rs/ruffle#8396] [General]
When i tried loading a game called urban basketball i got this message:
Unable to check top-level optout: Blocked a frame with origin

**Comment by n0samu** [comment]
That message is harmless and can be ignored. If you are having an issue with this content, it has nothing to do with the message that you mentioned. Please provide a link to the content if you'd like more information about any issue with the content.
